### ISSUE TITLE
Issue releasing

### ISSUE THREAD

**Issue by marcoscaceres** [highlightjs/highlight.js#1982]
@isagalaev I think I broke our ability to release with a recent commit. If you look at the [release failure](https://highlightjs.org/api/release/) - output below, it's missing the `demo/` directory.  

When updatehljs.py calls `./tools/build.js`, it needs to also have the `--docs` option passed to it. Is it possible to add that?  

```BASH
Traceback (most recent call last):
  File "/home/maniac/hljs_org/hljs_org/management/commands/updatehljs.py", line 104, in handle
    self._handle(version)
  File "/home/maniac/hljs_org/hljs_org/management/commands/updatehljs.py", line 79, in _handle
    call_command('publishtest')
  File "/home/maniac/hljs_org/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 130, in call_command
    return command.execute(*args, **defaults)
  File "/home/maniac/hljs_org/venv/lib/python3.6/site-packages/django/core/management/base.py", line 356, in execute
    output = self.handle(*args, **options)
  File "/home/maniac/hljs_org/hljs_org/management/commands/publishtest.py", line 44, in handle
    shutil.copytree(os.path.join(build_path, 'demo'), demo_dst)
  File "/usr/lib/python3.6/shutil.py", line 309, in copytree
    names = os.listdir(src)
FileNotFoundError: [Errno 2] No such file or directory: '/home/maniac/code/highlight/build/demo'
```

**Comment by isagalaev** [comment]
Yes, should be fairly easy. I think I'll be able to tackle it in about an hour.

**Comment by isagalaev** [comment]
Added the parameter, ran it manually, and it went well:

```
(venv) maniac@smorg:~/hljs_org$ ./manage.py updatehljs 9.15.1
2019-02-24 21:34:47,916 INFO hljs_org.updatehljs Checking out version 9.15.1...
2019-02-24 21:34:48,514 INFO hljs_org.updatehljs Checking version consistency within the source...
2019-02-24 21:34:48,516 INFO hljs_org.updatehljs Reinstalling node dependencies...
2019-02-24 21:35:11,502 INFO hljs_org.updatehljs Building CDN build to populate cache...
2019-02-24 21:35:16,971 INFO hljs_org.updatehljs Moving CDN build over to /home/maniac/app-data/hljs_cache
2019-02-24 21:35:16,988 INFO hljs_org.updatehljs Updating CDN repo at /home/maniac/code/cdn-release
2019-02-24 21:35:24,169 INFO hljs_org.updatehljs Tag 9.15.1 already exists in the local CDN repo
2019-02-24 21:35:26,453 INFO hljs_org.publishtest Publishing test page to /var/www/hljs_org/static
2019-02-24 21:35:26,453 INFO hljs_org.publishtest Building full browser build...
2019-02-24 21:35:32,009 INFO hljs_org.publishtest Copying highlight.pack.js...
2019-02-24 21:35:32,011 INFO hljs_org.publishtest Copying demo...
2019-02-24 21:35:32,027 INFO hljs_org.publishtest Building browser build for [python xml javascript http cpp sql clojure cs objectivec java swift css ruby makefile go coffeescript bash ini rust handlebars prolog typescript elm json]...
2019-02-24 21:35:34,299 INFO hljs_org.publishtest Copying highlight.site.pack.js...

0 static files copied to '/var/www/hljs_org/static', 63 unmodified.
2019-02-24 21:35:34,319 INFO hljs_org.updatehljs Updating news entry...
2019-02-24 21:35:35,292 INFO hljs_org.updatehljs Published npm version is 9.14.2
2019-02-24 21:35:35,293 INFO hljs_org.updatehljs Publishing version 9.15.1 to npm...
2019-02-24 21:35:42,974 INFO hljs_org.updatehljs Update to version 9.15.1 completed.
```
